src/utils/cacheHandlers.ts

```diff
@@ -0,0 +1,190 @@
+import { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { stringify } from 'query-string';
+
+export const DEFAULT_CACHE_CONFIG = { expiresIn: 1800 };
+const SENTINEL_HUB_CACHE = 'sentinelhub-v1';
+const EXPIRY_HEADER_KEY = 'Cache_Expires';
+
+export type CacheConfig = {
+  expiresIn: number;
+};
+
+export const fetchCachedResponse = async (request: any): Promise<any> => {
+  if (!isRequestCachable) {
```

Comment:
This check is always true.

Reply:
Thanks, missed this one. It has been fixed